### PR TITLE
fix: alter slogger so it can't throw errors

### LINKED ISSUE #5463: Slogger should not be able to cause consensus errors
## Describe the bug
https://github.com/Agoric/agoric-sdk/pull/5455 hit a strange deployment integration test failure: https://github.com/Agoric/agoric-sdk/runs/6633516162

The loadgen monitor fell out of consensus with the validators. The reason is that the validators were loading the otel telemetry, which had a bug, and contaminated the execution of a delivery.

## To Reproduce
See artifact from failed run

## Expected behavior
A slogging error should not contaminate execution

## Platform Environment
CI

## Screenshots
Swing store diff:
```diff
--- /dev/fd/63  2022-05-28 03:05:40.411104952 +0000
+++ /dev/fd/62  2022-05-28 03:05:40.411104952 +0000
@@ -33657,2 +33657,2 @@
-set local.v1.lastSnapshot {"snapshotID":"fed245bb264369a0ae25d925996800ac91f8637f7ce8b8e331d3b4fe293fa1fe","startPos":{"itemCount":202}}
-set local.snapshot.fed245bb264369a0ae25d925996800ac91f8637f7ce8b8e331d3b4fe293fa1fe ["v1"]
+set local.v1.lastSnapshot {"snapshotID":"50f2dfe3aaa9a98e47fc273dbf6a698538b03f08b4b5122fb7ec385a6660d5aa","startPos":{"itemCount":202}}
+set local.snapshot.50f2dfe3aaa9a98e47fc273dbf6a698538b03f08b4b5122fb7ec385a6660d5aa ["v1"]
@@ -70665 +70665 @@
-set acceptanceQueue [{"type":"notify","vatID":"v4","kpid":"kp756"}]
+set acceptanceQueue [{"type":"notify","vatID":"v4","kpid":"kp756"},{"type":"send","target":"ko443","msg":{"methargs":{"body":"[\"getAssetKind\",[]]","slots":[]},"result":"kp725"}}]
```

Slog diff:
```
55964,55965c55964,55967
< {"type":"console","source":"vat","state":"delivery","crankNum":4733,"vatID":"v3","deliveryNum":163,"level":"log","args":["WARNING: delivery from remote triggered error:",{"code":"SQLITE_CONSTRAINT_PRIMARYKEY"}]}
< {"type":"console","source":"vat","state":"delivery","crankNum":4733,"vatID":"v3","deliveryNum":163,"level":"log","args":["Error happened while processing \"[\"receive\",[\"87:74:deliver:ro+59:rp+48;[\\\"getAssetKind\\\",[]]\"]]\""]}
---
> {"type":"syscall","crankNum":4733,"vatID":"v3","deliveryNum":163,"syscallNum":24,"replay":false,"ksc":["send","ko443",{"methargs":{"body":"[\"getAssetKind\",[]]","slots":[]},"result":"kp725"}],"vsc":["send","o-92",{"methargs":{"body":"[\"getAssetKind\",[]]","slots":[]},"result":"p-139"}]}
> {"type":"syscall-result","crankNum":4733,"vatID":"v3","deliveryNum":163,"syscallNum":24,"replay":false,"ksr":["ok",null],"vsr":["ok",null]}
> {"type":"syscall","crankNum":4733,"vatID":"v3","deliveryNum":163,"syscallNum":25,"replay":false,"ksc":["subscribe","v3","kp725"],"vsc":["subscribe","p-139"]}
> {"type":"syscall-result","crankNum":4733,"vatID":"v3","deliveryNum":163,"syscallNum":25,"replay":false,"ksr":["ok",null],"vsr":["ok",null]}
```

### PR DESCRIPTION
**Pull request by FUDCo**
Fixes #5463

Note: this currently does not have any tests.  I was able to verify that it works by introducing errors with a temporary source mod to `controller.js` and then testing manually to eyeball that the appropriate stuff appeared in the console log, but this approach does not lend itself to automation.  However, I haven't as yet figured out a way to construct a test that could be used in CI.  Given the limited scope and relatively gentle nature of the changes as well as the very marginal edge case nature of the condition being dealt with here, it's not obvious to me that this one motivates a significant engineering investment to automate its testing.  That said, if anyone is inspired with a straightforward approach that will do the job, I'm all ears.

### CODE REVIEW

**Review by warner — APPROVED** [review]
Looks good, and yeah I agree manual testing is sufficient for this.
